src/celestia/qt/qtappwin.cpp

```diff
@@ -1059,52 +1059,88 @@ void CelestiaAppWindow::slotShowGLInfo()
     QString infoText;
     QTextStream out(&infoText, QIODevice::WriteOnly);
 
+    map<string, string> info;
+    m_appCore->getRenderer()->getInfo(info);
+
     // Get the version string
     // QTextStream::operator<<(const char *string) assumes that the string has
     // ISO-8859-1 encoding, so we need to convert in to QString
-    out << QString(_("<b>OpenGL version: </b>"));
-    const char* version = reinterpret_cast<const char*>(glGetString(GL_VERSION));
-    if (version != nullptr)
-        out << version;
-    else
-        out << "???";
-    out << "<br>\n";
+    if (info.count("API") > 0 && info.count("APIVersion") > 0)
```

Comment:
this block could be replaced by Helper::getRenderInfo I suppose?

Reply:
Qt uses HTML, while Helper::getRenderInfo provides plain text used by gtk & native win interfaces.